src/woo_publications/publications/archiving.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+from woo_publications.constants import ArchiveNominationChoices
+from woo_publications.metadata.models import InformationCategory
+
+
+def get_retention_informatie_category(
+    informatie_categorieen: QuerySet[InformationCategory],
+) -> InformationCategory | None:
+    """
+    Determine the most appropriate information category for archiving parameter derivation.
+
+    The following logic is applied:
+    - Information categories with :arg:`archiefnominatie` ``retain`` will always be used if present.
```

Comment:
```suggestion
    The following logic is applied:

    - Information categories with :arg:`archiefnominatie` ``retain`` will always be used if present.
```
the blank line is required for proper RST rendering I think.